updatehub-sdk/src/api/mod.rs

```diff
@@ -100,4 +97,37 @@ pub mod log {
         Debug,
         Trace,
     }
+
+    #[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
The `Entry` struct should go just after `Log` (so before the `Level` `enum`). I did the change prior to merging it.